dts/arm/silabs/efr32bg2x.dtsi

```diff
@@ -130,9 +130,11 @@
 			pstate_em1: em1 {
 				compatible = "zephyr,power-state";
 				power-state-name = "runtime-idle";
-				min-residency-us = <4>;
+				/* The minimum residency and exit latency is
+				 * managed by sl_power_manager on S2 devices. */
+				min-residency-us = <0>;
 				/* HFXO remains active */
-				exit-latency-us = <2>;
+				exit-latency-us = <0>;
```

Comment:
`0` is the default value for `min-residency-us` and `exit-latency-us` (see `PM_STATE_INFO_DT_INIT()`). You can drop these lines.